modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1598 @@
+For more information about how indexes impact query performance, see xref:indexes/search-performance-indexes/using-indexes.adoc[].
+
+These indexes can be viewed in the `owningConstraint` column returned by the xref:indexes/search-performance-indexes/managing-indexes.adoc#list-indexes[`SHOW INDEX`] command, and the `ownedIndex` column returned by the xref:constraints/managing-constraints.adoc#list-constraints[`SHOW CONSTRAINT`] command.
+
+.List constraints with backing indexes
+======
+
+.Query
+[source, cypher, test-exclude-cols=id]
+----
+SHOW CONSTRAINTS WHERE ownedIndex IS NOT NULL
+----
+
+.Result
+[source, queryresult]
+----
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| id | name                    | type                      | entityType     | labelsOrTypes  | properties                   | ownedIndex              | propertyType |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| 17 | "actor_fullname"        | "NODE_KEY"                | "NODE"         | ["Actor"]      | ["firstname", "surname"]     | "actor_fullname"        | NULL         |
+| 3  | "book_isbn"             | "UNIQUENESS"              | "NODE"         | ["Book"]       | ["isbn"]                     | "book_isbn"             | NULL         |
+| 7  | "book_title_year"       | "UNIQUENESS"              | "NODE"         | ["Book"]       | ["title", "publicationYear"] | "book_title_year"       | NULL         |
+| 13 | "director_imdbId"       | "NODE_KEY"                | "NODE"         | ["Director"]   | ["imdbId"]                   | "director_imdbId"       | NULL         |
+| 19 | "knows_since_how"       | "RELATIONSHIP_KEY"        | "RELATIONSHIP" | ["KNOWS"]      | ["since", "how"]             | "knows_since_how"       | NULL         |
+| 25 | "node_uniqueness_param" | "UNIQUENESS"              | "NODE"         | ["Book"]       | ["prop1"]                    | "node_uniqueness_param" | NULL         |
+| 15 | "ownershipId"           | "RELATIONSHIP_KEY"        | "RELATIONSHIP" | ["OWNS"]       | ["ownershipId"]              | "ownershipId"           | NULL         |
+| 9  | "prequels"              | "RELATIONSHIP_UNIQUENESS" | "RELATIONSHIP" | ["PREQUEL_OF"] | ["order", "author"]          | "prequels"              | NULL         |
+| 5  | "sequels"               | "RELATIONSHIP_UNIQUENESS" | "RELATIONSHIP" | ["SEQUEL_OF"]  | ["order"]                    | "sequels"               | NULL         |
++--------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+======
+
+.List indexes with owning constraints
+======
+
+.Query
+[source, cypher, test-exclude-cols=id]
```

Comment:
We use exclude-cols because the id column repeatedly caused test failures on our end. Not sure why the lastRead and readCount do not fail - maybe the test was built with those in mind. Will check. Regardless, it is not currently necessary.